notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb

```diff
@@ -0,0 +1,1333 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ! rm -rf {BUCKET_NAME}
</pre><p>This line could be dangerous for an existing bucket. I've seen in another notebook where we comment it out with a comment above "Uncomment the following line to delete the contents of the bucket". (I can update the notebook template BTW.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>